source/configure/site-configuration-settings.rst

```diff
@@ -1091,6 +1100,75 @@ Message priority
+
+.. config:setting:: posts-maxnumberofrecipientsforpersistentnotifications
+  :displayname: Maximum number of recipients for persistent notifications (Posts)
+  :systemconsole: Site Configuration > Posts
+  :configjson: .ServiceSettings.PersistentNotificationMaxRecipients
+  :environment: MM_SERVICESETTINGS_PERSISTENTNOTIFICATIONSMAXRECIPIENTS
+  :description: The maximum number of recipients users may send persistent notifications to. Default is **5**.
+
+Maximum number of recipients for persistent notifications
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
++---------------------------------------------------------------+-----------------------------------------------------------------------------------------+
+| The maximum number of recipients users may send persistent    | - System Config path: **Site Configuration > Posts**                                    |
+| notifications to.                                             | - ``config.json`` setting: ``.ServiceSettings.PersistentNotificationMaxRecipients: 5``  |
+|                                                               | - Environment variable: ``MM_SERVICESETTINGS_PERSISTENTNOTIFICATIONSMAXRECIPIENTS``     |
+| Numerical input. Default is **5**.                            |                                                                                         |
++---------------------------------------------------------------+-----------------------------------------------------------------------------------------+
+
+.. config:setting:: posts-frequencyofpersistentnotifications
+  :displayname: Frequency of persistent notifications (Posts)
+  :systemconsole: Site Configuration > Posts
+  :configjson: .ServiceSettings.PersistentNotificationIntervalMinutes
+  :environment: MM_SERVICESETTINGS_PERSISTENTNOTIFICATIONINTERVALMINUTES
+  :description: The number of minutes between repeated notifications for urgent messages sent with persistent notifications. Default is **5**.
+
+Frequency of persistent notifications
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
++---------------------------------------------------------------+-------------------------------------------------------------------------------------------+
+| The number of minutes between repeated notifications for      | - System Config path: **Site Configuration > Posts**                                      |
+| urgent messages sent with persistent notifications.           | - ``config.json`` setting: ``.ServiceSettings.PersistentNotificationIntervalMinutes: 5``  |
+|                                                               | - Environment variable: ``MM_SERVICESETTINGS_PERSISTENTNOTIFICATIONINTERVALMINUTES``      |
+| Numerical input. Default is **5**.                            |                                                                                           |
```

Comment:
Minimum is 2.

Also there may be an offset of up to 50% of the defined value before the first persistent notification is sent. (0/5 if this is important enough to mention)